scripts/setupDevelopmentShowcase.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+echo "Development script to add example application to the repository"
+mkdir example
+git clone git@github.com:aerogear/android-showcase-template.git
+cd android-showcase-template
+cp -a app/. ../example
```

Comment:
can we use a symlink here?

Reply:
Good idea, once we get the content of the showcase app moved over we can test a symlink over a full copy